src/components/Store.tsx

```diff
@@ -11,6 +11,7 @@ import RoomRenameProvider from './pages/Chat/RoomRenameProvider'
 
 export enum ActionType {
     AddMessages = 'add messages',
+    PublishMessage = 'publish message',
```

Comment:
We don't seem to be using this one any longer. Submitting messages goes through `useSend`.